tools/operator/launcher/dockercompose/network.go

```diff
@@ -60,9 +62,35 @@ func (d DockerCompose) DownLocalNetwork(config networkspec.Config) error {
 	if err != nil {
 		return err
 	}
+	err = d.removeChainCodeContainersAndImages()
+	if err != nil {
+		return err
+	}
 	return nil
 }
 
+func (d DockerCompose) removeChainCodeContainersAndImages() error {
```

Comment:
ok. I was thinking we would need this to be made available as a separate action, from test-input file. @suryalnvs may wish to use it for his upgrade test. Will that be simple addition, and still keep this here? (If we have to move it, then may I suggest we do it now?)

Reply:
Where exactly we move it @scottz64. I think this is the right position to place it as this file contains the removal of the docker network.

Reply:
Yes, it seems right. I was just thinking about when we will have to add an action too, to be able to call this function.

Reply:
Why we need to add an action @scottz64 . It will be a part of down network I guess.